internal/hrq/rqreconciler.go

```diff
@@ -147,7 +192,9 @@ func (r *ResourceQuotaReconciler) reportHRQEvent(ctx context.Context, log logr.L
 		return
 	}
 
-	for _, nnm := range r.getAncestorHRQs(inst) {
+	nnms := r.getAncestorHRQs(inst)
+
+	for _, nnm := range nnms {
```

Comment:
There are changes like this in several places. It seems that nnms is not used after this point. What is the purpose of this change?

In my personal opinion, upstream code changes should be kept to a minimum.

Reply:
> In my personal opinion, upstream code changes should be kept to a minimum.

I agree with you +1

Once it was after a lot of trial and error to change most functions to return an error. I have reviewed it all over, but please let me know if there are others.